pkg/workflows/wasm/host/module.go

```diff
@@ -100,6 +77,10 @@ type ModuleConfig struct {
 	IsUncompressed bool
 	Fetch          func(*wasmpb.FetchRequest) (*wasmpb.FetchResponse, error)
 
+	// Emitter is the function that will be called when the emit function is called in the WASM module.
+	// Implementation should emit to external consumer.
+	Emitter sdk.Emitter
```

Comment:
I don't think we should use the same interface for the guest and for the host. When we make the host method context aware, we'll need to start using different interfaces here, since the host emitter will accept a context, while the guest won't.